paimon-web-server/src/main/java/org/apache/paimon/web/server/data/vo/UserVO.java

```diff
@@ -41,7 +41,7 @@ public class UserVO {
 
     private String nickname;
 
-    private String userType;
+    private Integer userType;
```

Comment:
Could we introduce enum for user type?

Reply:
I think it's possible. I'll make some modifications